longitudinal_controller/src/pid.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python3
+
+
+class PID:
+    def __init__(
+        self,
+        gain_p: float = 0.0,
+        gain_i: float = 0.0,
+        gain_d: float = 0.0,
+    ):
+
+        self.kp = gain_p
+        self.ki = gain_i
+        self.kd = gain_d
+
+        self.output = 0
+        self.i_value = 0
+        self.previous_error = 0
+
+    def update(self, error: float, activate_anti_windup: bool, dt: float):
+        """
+        Function to calculate the controller's output given a set error.
+        An anti-windup strategy is implemented to avoid increasing the integral value if needed
+        Arguments:
+        - error [m/s]: Desired value - current value
+        - anti_windup: Boolean to enable the anti-windup strategy
+        - dt [s]: Time since last call
+        """
+
```

Comment:
Shall we put a check on `dt` and raise a warning if it is below a certain threshold (for instance 0.01s)?

Reply:
Done